docs/kratos/emails-sms/05_custom-email-templates.mdx

```diff
@@ -13,7 +13,12 @@ The system uses templates in these flows:
    - `recovery.valid` - the system sends these messages when the user completes the recovery flow successfully.
    - `recovery.invalid` - the system sends these messages when the recovery flow fails, for example when you start the recovery
      flow using an email address that isn't associated with a registered user account.
-2. `verification` - the system sends these messages when users verify their accounts after signing up.
+1. `recovery_code` - the system sends these messages when users recover their accounts, for example after forgetting their
+   password.
+   - `recovery_code.valid` - the system sends these messages when the user provided a valid recovery address.
+   - `recovery_code.invalid` - the system sends these messages when the recovery flow fails, for example when you start the
+     recovery flow using an email address that isn't associated with a registered user account.
+1. `verification` - the system sends these messages when users verify their accounts after signing up.
```

Comment:
I think that here you should make a distinction between `recovery` and `recovery_code` bc right now you have the same description for two different templates. 

I'd do it by editing the desc. slightly: 

```
the system sends these messages when users recover their accounts, for example after forgetting their
   password. This template is used when the recovery strategy is set to `code`
```

^ and the same for `recovery` (strategy set to `link`) and we're golden.


Also, can you please keep increasing numbers in the ordered list instead of doing all `1.`? I know it's still rendered properly, but it's just more natural to work with when you stick to the "classic" approach. Thanks!

Reply:
I've worded it a bit better and also removed the "for example" block. 

>Also, can you please keep increasing numbers in the ordered list instead of doing all 1.? I know it's still rendered properly, but it's just more natural to work with when you stick to the "classic" approach. Thanks!

It was that way before, just added my block :) 
The advantage of 1) is btw. that if you add another block, you don't need to redo the numbers. VSCode (with proper plugins) now does it automatically, but other editors might not. Just FYI.